src/st/dai/e_breakable.c

```diff
@@ -40,14 +86,18 @@ static u8 draw_modes[] = {
     DRAW_TPAGE | DRAW_TPAGE2,
     DRAW_DEFAULT,
     DRAW_DEFAULT};
-static u16 hitbox_offsets[] = {0, 0, -24, -16, 0, 0, 0, 0, 0, 0, 0, 0};
-static s16 debris_offsets[] = {0, 1, 2, 2, 3, 0, 1, 2, 3, 0};
+static u16 hitbox_offsets_y[] = {0, 0, -24, -16, 0, 0, 0, 0, 0, 0, 0, 0};
+/* End of arrays indexed by breakableType */
```

Comment:
Is there any way to make this more readable?
I tried using a struct like `ObjInit`/`BackgroundBlockInit`, but I couldn't get it to work.

Reply:
it's fine to keep it like that for now